mcs_benchmark_data/models/benchmark_sample.py

```diff
@@ -22,7 +22,7 @@
 class BenchmarkSample(_Model):
     """An entry in a benchmark dataset"""
 
-    includedInDataset: str
+    datasetURI: URIRef
```

Comment:
Fields should be snake_case.
dataset_uri
question_type
question_category
antecedent